lentille-gitlab/lentille-gitlab.cabal

```diff
@@ -42,6 +42,7 @@ library
                      , morpheus-graphql-client    ^>= 0.17
   exposed-modules:     Lentille.GitLab
                      , Lentille.GitLab.MergeRequests
+                     , Lentille.GitLab.Transformer
```

Comment:
Transformer usually mean something else. Would using `Adapter` be more appropriate?